int64 agv_command
---
int64 niryo_command
